app/assets/stylesheets/ama_layout/layout_components/callouts.scss

```diff
@@ -9,6 +9,7 @@
     @include callout($brand-green);
     margin-bottom: $base-margin;
     color: $white;
+    padding: $base-padding;
```

Comment:
I'm not sure if you need the $base-padding class here, the @include callout($brand-green) should have padding on it already.